pallets/utxo/src/lib.rs

```diff
@@ -377,19 +379,22 @@ pub mod pallet {
         }
 
         /// Create a new output to call a smart contract routine.
-        pub fn new_call_pp(
-            value: Value,
-            dest_account: AccountId,
-            fund: bool,
-            input: Vec<u8>,
-        ) -> Self {
+        pub fn new_call_pp(value: Value, dest_account: AccountId, input: Vec<u8>) -> Self {
             Self {
                 value,
                 destination: Destination::CallPP(dest_account, fund, input),
```

Comment:
How does this compile? The `fund` parameter is not defined and also is no longer part of the `CallPP` variant

Reply:
This is an outdated diff and the build works with latest changes